### DISCUSSION TITLE
请问如何做云同步

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2180] [Q&A]
我有群晖 支持目录同步 也支持webdav
我目前的法子是 同步了整个笔记本的目录，同时也同步了vnote的安装目录
然后盘符都是一样的 就这样用

但我总感觉不大对
最大的问题是快捷键 ctrl + g 不管用 但 ctrl+f 又是可以的我不知道咋回事 是不是因为这个原因
想看看各位大佬是如何同步的

**Comment by Binohow** [comment]
我直接用的坚果云，同步笔记文件夹

**Reply by tamlok** [reply]
ctrl-g那个还是没说明白吗？`Ctrl+G, E`才是一个完整的快捷键，`Ctrl+G`只是其中的前导键，当然按了没反应。你认真读一下快捷键说明呀。

同步的话，自己随便找一个第三方的服务把整个笔记本根文件夹同步好就可以了。
